driver/razercore_driver.c

```diff
@@ -455,6 +455,10 @@ static ssize_t razer_attr_read_device_mode(struct device *dev, struct device_att
  * Write only is 0220
  * Read and write is 0664
  */
+#if defined(WIN32) || defined(_WIN64)
+#undef DEVICE_ATTR
```

Comment:
Would it not make sense to define this somewhere like razercommon.h? then it would only need to be done once right?